scripts/insert_into_appointments.sql

```diff
@@ -1 +1,37 @@
 -- insert appointments creating connections between doctors and patients
+
+INSERT INTO appointments(date, time, doctor_id, patient_id)
```

Comment:
This won't run because you named your columns `start_date` and `start_time`, but you're trying to insert into the `date` and `time` columns, which don't exist.